src/core/url_utils.ts

```diff
@@ -47,14 +46,13 @@ export function getStudyRawConfigUrl(
   study: string,
   seedType: SeedType,
 ): string {
-  const BRAVE_VARIATIONS_REPO_PATTERN = '^github\\.com/brave/brave-variations$';
   if (seedType === SeedType.UPSTREAM)
     return `${getGitHubStorageUrl()}/blob/main/study/all-by-name/${study}`;
-  const branch = seedType === SeedType.MAIN ? 'main' : 'staging';
-  return makeSourceGraphUrl(
```

Comment:
Looks like https://sourcegraph.com stopped indexing brave-variations, so I've switched to github.
Also, the new URL adds `studies/*.json5` to the file pattern.